lib/DDG/Spice/SublimePackages.pm

```diff
@@ -24,8 +23,16 @@ my $skip = join "|", share('skipwords.txt')->slurp(chomp => 1);
 
 handle remainder => sub {
     return unless $_; 
+    # OS search filters
+    s/linux/:linux/g; 
+    s/windows/:win/g;
+    s/mac\s?os\s?x|os\s?x/:osx/g;
+    # Version search filters
+    s/version 2/:st2/g;
+    s/version 3/:st3/g;
```

Comment:
@moollaza I've set these to only match exactly `version [2-3]`, maybe this could be expanded?